doc/users/next_whats_new/separated_hatchcolor.rst

```diff
@@ -0,0 +1,58 @@
+Separated ``hatchcolor`` from ``edgecolor``
+-------------------------------------------
+
+`~matplotlib.patches.Patch` gained a new *hatchcolor* parameter to explicitly
+control hatch colors. Previously, hatch colors were the same as edge colors,
+with a fallback to :rc:`hatch.color` if the patch did not have an edge color.
+
+Fallback Logic
+~~~~~~~~~~~~~~
+When the *hatchcolor* parameter is specified, it will be used for the hatch.
+If it is not specified, it will fallback to using :rc:`hatch.color`.
+
+The special value 'edge' takes over the patch edgecolor, with a fallback to
```

Comment:
```suggestion
The special value 'edge' uses the patch edgecolor, with a fall back to
```

Reply:
Oops, my mistake; the `fallback` here is a noun, not a verb, and shouldn't get the space added.